tests/integration/ipfs.test.js

```diff
@@ -0,0 +1,20 @@
+import { writeToIPFS, dereferenceFromIPFS } from '../../src/utils/rdf';
+
+const rdfInput = `
+  @prefix : <http://example.org/stuff/1.0/> .
+  :a :b ( "apple" "banana" ) .
+`;
+
+const rdfCID = 'QmQeskBk9TsdWXaNX26PvuzkxU8zzoWZo5dmJMPuw5vcSS';
+
+describe('IPFS', () => {
+  test('Can write document to IPFs', async () => {
+    const cid = await writeToIPFS(rdfInput);
+    expect(cid.toString()).toEqual(rdfCID);
+  }, 10000);
+
+  test('Can dereference document from IPFs', async () => {
```

Comment:
Test 2 implicitly depends on test 1.
```suggestion

```

Reply:
yes it does, and i don't see how your suggestion improves that or why its an issue. your suggestion only makes it harder to see what goes wrong, imo

Reply:
was able to reduce the tests by remove write method, assuming that is now covered by ipfs-http-client library tests